src/main/app/fees/feesClient.ts

```diff
@@ -34,49 +35,48 @@ export class FeesClient {
+   * @returns {Promise.<FeeOutcome>} promise containing the Fee outcome (including fee amount in GBP)
    */
-  static async calculateFee (feeCode: string, amount: number): Promise<CalculationOutcome> {
-    ClaimValidator.claimAmount(amount)
-    const amountInPennies = MoneyConverter.convertPoundsToPennies(amount)
-    if (amountInPennies <= 0) {
-      throw new Error(`Amount must be at least 1 penny, amount was: ${amountInPennies}`)
+  static async calculateFee (eventType: string, amount: number, channel: string): Promise<FeeOutcome> {
+    if (StringUtils.isBlank(eventType)) {
+      throw new Error('Fee eventType is required')
     }
-
-    const fee: object = await request.get(`${feesUrl}/range-groups/${feeCode}/calculations?value=${amountInPennies}`)
-    return plainToClass(CalculationOutcome, fee)
+    ClaimValidator.claimAmount(amount)
+    const feeUri: string = `${feesUrl}/fees-register/fees/lookup?service=${service}&jurisdiction1=${jurisdiction1}&jurisdiction2=${jurisdiction2}&channel=${channel}&event=${eventType}&amount_or_volume=${amount}`
+    const fee: object = await request.get(feeUri)
+    return plainToClass(FeeOutcome, fee)
   }
 
   /**
    * Get the issue fee range group
-   * @returns {Promise.<RangeGroup>} promise containing the range group (including fee amounts in pennies)
+   * @returns {Promise.<FeeRange>} promise containing the range group (including fee amounts in GBP)
    */
-  static async getIssueFeeRangeGroup (): Promise<RangeGroup> {
-    return this.getRangeGroup(issueFeeCode)
+  static async getIssueFeeRangeGroup (): Promise<FeeRange[]> {
+    return this.getRangeGroup(issueFeeEvent, onlineChannel)
   }
 
   /**
    * Get hearing fee range group
-   * @returns {Promise.<RangeGroup>} promise containing the range group (including fee amounts in pennies)
+   * @returns {Promise.<FeeRange>} promise containing the range group (including fee amounts in GBP)
    */
-  static async getHearingFeeRangeGroup (): Promise<RangeGroup> {
-    return this.getRangeGroup(hearingFeeCode)
+  static async getHearingFeeRangeGroup (): Promise<FeeRange[]> {
+    return this.getRangeGroup(hearingFeeEvent, defaultChannel)
   }
 
-  private static async getRangeGroup (code: string): Promise<RangeGroup> {
-    if (StringUtils.isBlank(code)) {
-      throw new Error('Fee code is required')
+  private static async getRangeGroup (eventType: string, channel: string): Promise<FeeRange[]> {
+    if (StringUtils.isBlank(eventType)) {
+      throw new Error('Fee eventType is required')
     }
-    const range: object = await request.get(`${feesUrl}/range-groups/${code}`)
-    return plainToClass(RangeGroup, range)
+    const feeUriForRange: string = `${feesUrl}/fees-register/fees?service=${service}&jurisdiction1=${jurisdiction1}&jurisdiction2=${jurisdiction2}&channel=${channel}&event=${eventType}&feeVersionStatus=approved`
```

Comment:
To make it shorter we could just name it `uri`, in this context there will be just one URI and it should be clear which one is used.